src/Nethermind/Nethermind.Runner/Hive/mainnet.cfg

```diff
@@ -1,29 +0,0 @@
-{
```

Comment:
We don't need special Hive config?

Reply:
We have hive.cfg, this duplicate of mainnet.cfg doesn't seem useful. Tests are running well without it